lms/djangoapps/teams/tests/test_views.py

```diff
@@ -147,6 +147,49 @@ def test_bad_course_id(self):
         response = self.client.get(bad_team_url)
         self.assertEqual(404, response.status_code)
 
+    def get_user_course_specific_teams_list(self):
+        """Gets the list of user course specific teams."""
+
+        # Create a course two
+        course_two = CourseFactory.create(
+            teams_configuration={
+                "max_team_size": 1,
+                "topics": [
+                    {
+                        "name": "Test topic for course two",
+                        "id": 1,
+                        "description": "Description for test topic for course two."
+                    }
+                ]
+            }
+        )
+
+        # Login and enroll user in both course course
+        self.client.login(username=self.user.username, password=self.test_password)
+        CourseEnrollmentFactory.create(user=self.user, course_id=self.course.id)
+        CourseEnrollmentFactory.create(user=self.user, course_id=course_two.id)
+
+        # Create teams in both courses
+        course_one_team = CourseTeamFactory.create(name="Course one team", course_id=self.course.id, topic_id=1)
+        course_two_team = CourseTeamFactory.create(name="Course two team", course_id=course_two.id, topic_id=1)  # pylint: disable=unused-variable
+
+        # Check that initially list of user teams in course one is empty
+        course_one_teams_url = reverse('teams_dashboard', args=[self.course.id])
+        response = self.client.get(course_one_teams_url)
+        self.assertIn('"teams": {"count": 0', response.content)
```

Comment:
sorry, I didn't realize we were expecting JSON here! Why don't you do:
`team_count = json.loads(response.content)['teams']['count']`
and then check `team_count` is what you expect it to be?